cabal.project

```diff
@@ -59,6 +53,15 @@ package cardano-wallet-launcher
 package cardano-wallet-core-integration
   optimization: False
 
+source-repository-package
```

Comment:
There are probably quite a few Haskell packages we can prune from here

Reply:
Yes I know, but I wanted to leave that for a second step, so I can first unblock PR's that required the Playground to be in its own repository